doc/stdlib/math.rst

```diff
@@ -164,11 +172,29 @@ Mathematical Operators
 
    The floored quotient and modulus after division. Equivalent to ``(fld(x,y), mod(x,y))``\ .
 
+.. function:: fld1(x,m)
```

Comment:
So do I usually, but here I thought that this would make the tuple less readable.